src/sql/src/plan/plan_utils.rs

```diff
@@ -20,18 +20,14 @@ use crate::plan::PlanError;
 /// Renames the columns in `desc` with the names in `column_names` if
 /// `column_names` is non-empty.
 ///
-/// Returns an error if the length of `column_names` is not either zero or the
-/// arity of `desc`.
+/// Returns an error if the length of `column_names` is greater than the arity
+/// of `desc`.
 pub fn maybe_rename_columns(
     context: impl fmt::Display,
-    desc: RelationDesc,
+    desc: &mut RelationDesc,
```

Comment:
just curious: why this additional change of using `desc` as a mutable in/out parameter?